lib/puppet/type/sensu_secret.rb

```diff
@@ -0,0 +1,93 @@
+require_relative '../../puppet_x/sensu/type'
+require_relative '../../puppet_x/sensu/array_property'
+require_relative '../../puppet_x/sensu/array_of_hashes_property'
+require_relative '../../puppet_x/sensu/hash_property'
+require_relative '../../puppet_x/sensu/integer_property'
+
+Puppet::Type.newtype(:sensu_secret) do
+  desc <<-DESC
+@summary Manages Sensu Secrets
+@example Add a secret to default namespace
```

Comment:
`Manage a secret in` as opposed to `Add a secret to`, is a way to describe what's happening in an idempotent fashion.

Reply:
Wording updated